flow/project.py

```diff
@@ -491,13 +516,13 @@ def __init__(self, pre=None, post=None):
     def __str__(self):
         return "{type}(cmd='{cmd}')".format(type=type(self).__name__, cmd=self._cmd)
 
-    def eligible(self, job, ignore_conditions=IgnoreConditions.NONE):
+    def eligible(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
         """Eligible, when all pre-conditions are true and at least one post-condition is false,
         or corresponding conditions are ignored.
-        :param job:
-            The signac job handles.
-        :type job:
-            :class:`~signac.contrib.job.Job`
+        :param jobs:
```

Comment:
The parameter is actually `*jobs`, not `jobs`. However, if you just add an asterisk, Sphinx will interpret the asterisk as "beginning bold text." This is annoying, but you can fix it by changing this to a raw string (`r"""`) and escaping the asterisk like this:

```
r"""Docstring header.

:param \*jobs:
    A list of signac job handles.
:type \*jobs:
    list of :class:`~signac.contrib.job.Job`
"""
```
Please use that pattern in all docstrings accepting `*jobs`.